dispatcher/backend/src/common/schemas/offliners/devdocs.py

```diff
@@ -0,0 +1,162 @@
+from marshmallow import fields
+
+from common.schemas import SerializableSchema, String
+from common.schemas.fields import (
+    validate_output,
+    validate_zim_description,
+    validate_zim_longdescription,
+)
+
+
+class DevDocsFlagsSchema(SerializableSchema):
+    class Meta:
+        ordered = True
+
+    all_flag = fields.Boolean(
```

Comment:
Is the `skip_slig_regex` applied when using this? That's unclear

Reply:
Good point